configs/project-refs.conf

```diff
@@ -139,7 +139,7 @@ vars.uris: {
   scala-xml-quote-uri:          "https://github.com/allanrenucci/scala-xml-quote.git#bintray"
   scala-xml-uri:                "https://github.com/scala/scala-xml.git#1.x"
   scalacheck-shapeless-uri:     "https://github.com/alexarchambault/scalacheck-shapeless.git#294ff5ce33dddc82"  # was master
-  scalacheck-uri:               "https://github.com/scalacommunitybuild/scalacheck.git#community-build-2.13"  # was rickynils, master; was ricknyils, 1.14.0_sonatype
+  scalacheck-uri:               "https://github.com/rickynils/scalacheck.git"
```

Comment:
I compared the community-build branch with current master, and there are few if any changes, now.